pkg/model/serializer.go

```diff
@@ -742,13 +765,21 @@ func (d *Dev) UnmarshalYAML(unmarshal func(interface{}) error) error {
 				Command     Command           `json:"command,omitempty" yaml:"command,omitempty"`
 				Reverse     []Reverse         `json:"reverse,omitempty" yaml:"reverse,omitempty"`
 				Mode        string            `json:"mode,omitempty" yaml:"mode,omitempty"`
+
+				UnsupportedFields map[string]interface{} `yaml:",inline" json:"-"`
 			}
 
 			hybridModeDev := &hybridModeInfo{}
 			err := unmarshal(hybridModeDev)
 			if err != nil {
 				return err
 			}
+
+			for _, field := range hybridUnsupportedFields {
```

Comment:
Could we iterate over `hybridModeDev.UnsupportedFields[field]`. That way we don't need the list of unsupported fields which does not scale and if we add a new one we may forget to add it to the list.

```suggestion
for field, _ := range hybridModeDev.UnsupportedFields {
    oktetoLog.Warning("In hybrid mode, the field '%s' specified in your manifest is ignored", field)
}
```

Reply:
addressed in https://github.com/okteto/okteto/pull/3764/commits/9b62488e14fd06420b5ef33a49f1b6cff9ea9194#diff-4f6045f32d12e1bf0cf56de0a6ec8e8937dca49ea38367f0f9dbf11e359ff4c9R778